omnipaxos/src/omni_paxos.rs

```diff
@@ -295,15 +311,13 @@ where
     }
 }
 
-/// An error returning the proposal that was failed due to that the current configuration is stopped.
+/// An error indicating a failed proposal due to the current configuration being already stopped
+/// or due to an invalid proposed configuration.
 #[allow(missing_docs)]
 #[derive(Debug)]
-pub enum ProposeErr<T>
-where
-    T: Entry,
-{
-    Normal(T),
-    Reconfiguration(ClusterConfig),
```

Comment:
I think the naming/documentation was not good before. But a ProposeErr returns what was failed to be proposed, and I want users to be able to get back the `T` that failed to get proposed.